apis/r/R/RcppExports.R

```diff
@@ -105,17 +105,17 @@ check_arrow_array_tag <- function(xp) {
 #' summary(rl)
 #' }
 #' @export
-sr_setup <- function(ctx, uri, colnames = NULL, qc = NULL, dim_points = NULL, dim_ranges = NULL, config = NULL, loglevel = "auto") {
-    .Call(`_tiledbsoma_sr_setup`, ctx, uri, colnames, qc, dim_points, dim_ranges, config, loglevel)
+sr_setup <- function(uri, config, colnames = NULL, qc = NULL, dim_points = NULL, dim_ranges = NULL, loglevel = "auto") {
+    .Call(`_tiledbsoma_sr_setup`, uri, config, colnames, qc, dim_points, dim_ranges, loglevel)
 }
 
-#' @rdname sr_setup
+#' @rdname sr_complete
```

Comment:
sr_setup is a forward reference. If you are going through the trouble of renaming what the man page is, should we go all in and and set `@rd_name` to something meaningful like 'soma_iterated_reader' ?

Reply:
no worries, my misunderstand -- will revert